docs/UsersGuide.asciidoc

```diff
@@ -159,6 +159,13 @@ and/or os-autoinst's own behavior are:
 * +QEMUVGA+ parameter to declare the video hardware configuration in QEMU
 
 
+==== Test Suite extension
+
+Test suites can extend existing test suites. Specify the test suite you want
+to extend in the variable +EXTENDS+. All variables are inherited from the
```

Comment:
Hopefully nobody is using that variable already for other purposes.

Reply:
nope, we checked on osd :)